napari/layers/points/_points_mouse_bindings.py

```diff
@@ -0,0 +1,95 @@
+import numpy as np
+
+from ._points_utils import points_in_box
+
+
+def select(layer, event):
```

Comment:
can `layer._is_selecting`, `layer._drag_start`, and `layer._drag_box` be made local to this function?

Reply:
unfortunately right now they are all also used inside the main model file, either in the `_move` function, which could probably be refactored into this file or inside some of the highlighting stuff. Maybe we leave this for now, and possibly revisit after we're done some of the interaction box stuff in #999